modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2NewKeyGenerationPortal.testcase

```diff
@@ -163,6 +135,98 @@ definition {
 		}
 	}
 
+	@description = "Automation Ticket: LPS-151337 | Test Summary: Verify that after filling out all the fields the Next button is enabled."
+	@ignore = "true"
+	@priority = 4
+	test AssertComplimentaryPortalKeyCanBeCreated {
+		
+		task ("Given Active the option in koroneiki 'Allow Self Provissioning'") {
+			JSONCP.activeAllowSelfProvisioning(koroneiki = "KOR-3809080");
+		}
+
+		task ("And the user Role is Changed") {
+			CP2Utils.assigneToRoleViaUsersTab(
+				searchItem = "test",
+				accountRoleTitle = "Administrator",
+				accountUserName = "Test Test",
+				testAccount = "2022 FIFA World Cup Qatar - FAN ID");
+		}
+
+		task ("And an user go to Customer Portal") {
+			ApplicationsMenu.gotoSite(key_site = ${testSiteName});
+		}
+
+		task ("And enter in test account") {
+			
+			CustomerPortal.search(searchItem = "2022 FIFA World Cup Qatar - FAN ID");
+			
+			CustomerPortal.enterOnProject(title = "2022 FIFA World Cup Qatar - FAN ID");
+		}
+			if (IsElementPresent(key_text = "Start Project Setup", locator1 = "Button#ANY")) {
+			CP2Utils.onboardingFlow();
+		}
+		else {
+
+			task ("And click on Portal product") {
+				CP2NavigationMenu.goToProductActivation();
+
+				CP2NavigationMenu.clickOnProductButton(product = ${testProduct});
+			}
+		}
+
+		task ("And go to generate a new key") {
+			CP2ActivationKeys.generateNewKey();
+		}
+
+		task ("When fulfill first form for Generate New License") {
```

Comment:
Change to "When filling in the first form to Generate New License"

Reply:
Good suggestion, I will change

Reply:
Done!